SingularityExecutor/src/main/java/com/hubspot/singularity/executor/task/SingularityExecutorArtifactFetcher.java

```diff
@@ -194,7 +194,7 @@ private void downloadRemoteArtifact(RemoteArtifact remoteArtifact, ArtifactManag
       if (Objects.toString(fetched.getFileName()).endsWith(".tar.gz")) {
         artifactManager.untar(fetched, task.getArtifactPath(remoteArtifact, task.getTaskDefinition().getTaskDirectoryPath()));
       } else {
-        artifactManager.copy(fetched, task.getArtifactPath(remoteArtifact, task.getTaskDefinition().getTaskAppDirectoryPath()), remoteArtifact.getFilename());
+        artifactManager.copy(fetched, task.getArtifactPath(remoteArtifact, task.getTaskDefinition().getTaskDirectoryPath()), remoteArtifact.getFilename());
```

Comment:
is this correct in every circumstance? I think we need to combine getTaskDirectoryPath() and targetFolderRelativeToTask() (if present)

Reply:
nevermind, I forgot that's already happening in `getArtifactPath()`